code/modules/jobxp/JobXPRewards.dm

```diff
@@ -244,6 +244,21 @@ mob/verb/checkrewards()
 	required_levels = list("Botanist"=10)
 	path_to_spawn = /obj/item/reagent_containers/glass/wateringcan/rainbow
 
+/datum/jobXpReward/botanist/jumpsuit
+	name = "Senior Botanist Jumpsuit"
+	desc = "An old jumpsuit with an earthy smell to it."
+	required_levels = list("Botanist"=15)
+	icon_state = "?"
+	claimable = 1
+	claimPerRound = 1
+
+	activate(var/client/C)
+		boutput(C, "The jumpsuit pops into existance!")
+		var/obj/item/I = new/obj/item/clothing/under/misc/hydroponics()
+		I.set_loc(get_turf(C.mob))
```

Comment:
```suggestion
		var/obj/item/I = new/obj/item/clothing/under/misc/hydroponics(get_turf(C.mob))
```
For most things including a location as the first argument to a `new` call will spawn that thing at that location. Works here so we can cut a line out.